ontology/account/account.py

```diff
@@ -20,75 +21,59 @@
 from ontology.crypto.signature_scheme import SignatureScheme
 from ontology.crypto.signature_handler import SignatureHandler
 
+_SHA256 = SignatureScheme.SHA256withECDSA
+_SHA3_224 = SignatureScheme.SHA3_224withECDSA
+_SHA3_384 = SignatureScheme.SHA3_384withECDSA
+_SHA512 = SignatureScheme.SHA512withECDSA
+
+SUPPORTED_SCHEMES = [_SHA256, _SHA3_224, _SHA3_384, _SHA512]
```

Comment:
Could you explain to me why this change is necessary? In fact, more signature scheme will be supported in the future. I think this simplification will lead to confusion in the future.